src/test/scala/temple/builder/DatabaseBuilderTestData.scala

```diff
@@ -73,4 +75,102 @@ object DatabaseBuilderTestData {
         ),
       ),
     )
+
+  val sampleInsertStatement: Statement = Insert(
+    "test-service",
+    Seq(
+      Assignment(Column("id"), PreparedValue),
+      Assignment(Column("bankBalance"), PreparedValue),
+      Assignment(Column("name"), PreparedValue),
+      Assignment(Column("isStudent"), PreparedValue),
+      Assignment(Column("dateOfBirth"), PreparedValue),
+      Assignment(Column("timeOfDay"), PreparedValue),
+      Assignment(Column("expiry"), PreparedValue),
+      Assignment(Column("image"), PreparedValue),
+    ),
+    returnColumns = Seq(
+      Column("id"),
+      Column("bankBalance"),
+      Column("name"),
+      Column("isStudent"),
+      Column("dateOfBirth"),
+      Column("timeOfDay"),
+      Column("expiry"),
+      Column("image"),
+    ),
+  )
+
+  val sampleReadStatement: Statement = Read(
+    "test-service",
+    Seq(
+      Column("id"),
+      Column("bankBalance"),
+      Column("name"),
+      Column("isStudent"),
+      Column("dateOfBirth"),
+      Column("timeOfDay"),
+      Column("expiry"),
+      Column("image"),
+    ),
+    Some(PreparedComparison("id", ComparisonOperator.Equal)),
+  )
+
+  val sampleUpdateStatement: Statement = Update(
+    "test-service",
+    Seq(
+      Assignment(Column("id"), PreparedValue),
```

Comment:
Lol, I wonder if we should make `PreparedValue` the default argument for `Assignment`?

Reply:
Just a suggestion, seems like it's all we use